presto-spi/src/main/java/com/facebook/presto/spi/function/FunctionMetadata.java

```diff
@@ -38,6 +38,7 @@
     private final FunctionImplementationType implementationType;
     private final boolean deterministic;
     private final boolean calledOnNullInput;
+    private final Optional<String> version;
```

Comment:
Do we wanna create a specify class called FunctionVersion and make a const `DEFAULT_VERSION`? Then we can avoid optional or string

Reply:
I was thinking it's probably preferable to not show version for builtin functions. If we have a `DEFAULT_VERSION` then do we check for that and not displaying it or do we always display the default version?

Reply:
Or maybe just create a `FunctionVersion` class wrapping `Optional<String>`. It could be easier in the future in case we might want to do something to it.